package/upgrade/extra_manifests/longhorn-settings-default-data-path.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: longhorn.io/v1beta1
+kind: Setting
+metadata:
+  name: default-data-path
+  namespace: longhorn-system
+value: /var/lib/longhorn_data
```

Comment:
Should this be adjusted to `/var/lib/harvester/defaultdisk` as well?

Reply:
Yes we should, thanks!